xbmc/video/ContextMenus.cpp

```diff
@@ -119,7 +120,17 @@ static void SetPathAndPlay(CFileItem& item)
     item.SetPath(item.GetVideoInfoTag()->m_strFileNameAndPath);
   }
   item.SetProperty("check_resume", false);
-  g_application.PlayMedia(item, "", PLAYLIST_NONE);
+
+  CFileItemPtr movieItem((std::make_shared<CFileItem>(item)));
```

Comment:
IMO `std::make_shared<>()` should always be preferred if possible. It's more efficient and exception-safe.